linera-service/src/linera.rs

```diff
@@ -848,6 +848,36 @@ enum ClientCommand {
         balance: Amount,
     },
 
+    /// Subscribes to a system channel.
+    Subscribe {
+        /// Chain id (must be one of our chains).
+        #[arg(long = "subscriber")]
+        subscriber: Option<ChainId>,
+
+        /// Chain id (must be one of our chains).
+        #[arg(long = "publisher")]
```

Comment:
nit: I believe you don't need `= "publisher"`

Reply:
Cool, btw should we also remove `= "subscriber"`

Reply:
Yes, it's only needed if you want the argument to have a different name than the field:
https://docs.rs/clap/latest/clap/_derive/index.html#arg-attributes

Reply:
Cool thanks! That helps